model-optimizer/mo/utils/check_ie_bindings.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from version import get_mo_version
+from extract_release_version import extract_release_version
+
+
+def try_to_import_ie():
+    try:
+        from openvino.inference_engine import IECore, get_version
+        from openvino.offline_transformations import ApplyMOCTransformations
+
+        ie_version = get_version()
+        mo_version = get_mo_version()
+
+        if mo_version not in ie_version:
```

Comment:
Why not check `mo_version != ie_version`?

Reply:
I'll add description.

Reply:
Changed to regexp.